src/client/sandbox/shadow-ui.js

```diff
@@ -116,6 +118,19 @@ export default class ShadowUI extends SandboxBase {
         };
     }
 
+    static _markElementCollections (el) {
+        Object.defineProperty(el.children, IS_SHADOW_CONTAINER_COLLECTION_FLAG, {
+            configurable: false,
```

Comment:
`false` is a default value.
You can write as `Object.defineProperty(el.children, IS_SHADOW_CONTAINER_COLLECTION_FLAG, { value: true })`